aerialist/px4/drone_test.py

```diff
@@ -219,6 +224,7 @@ def __init__(
         self.headless = headless
         self.obstacles: List[Obstacle] = obstacles
         self.home_position = home_position
+        self.world_file_name = None
```

Comment:
what is this compared to the "world" above? i think you can use the same variable, it is not much used yet as far as i remember